node/network/protocol/src/lib.rs

```diff
@@ -404,4 +411,12 @@ pub mod v1 {
 	}
 
 	impl_try_from!(CollationProtocol, CollatorProtocol, CollatorProtocolMessage);
+
+	/// Get the payload that should be signed and included in a `Declare` message.
+	///
+	/// The payload is the local peer id of the node, which serves to prove that it
+	/// controls the collator key it is declaring an intention to collate under.
+	pub fn declare_signature_payload(peer_id: &sc_network::PeerId) -> Vec<u8> {
+		peer_id.to_bytes()
```

Comment:
Should we include some magic in this payload?

Reply:
What kind of magic? :)

Reply:
`b"COLL" or something?